libraries/AP_HAL_ChibiOS/hwdef/Pixhawk6/hwdef-bl.dat

```diff
@@ -15,7 +15,7 @@ USB_STRING_PRODUCT "Pixhawk6-BL"
 USB_STRING_SERIAL  "%SERIAL%"
 
 # board ID for firmware load
-APJ_BOARD_ID 139
```

Comment:
pixhawk6 doesn't need a renumber, as "PIXHAWK4PRO" and "Pixhawk6" are the same board

Reply:
we don't want to use the generic FMUv6 as that will lead to the same issues we had with fmuv5